src/game.cpp

```diff
@@ -2264,6 +2264,56 @@ bool Game::getServerContent(bool *aborted)
 				client->nodedefReceived()) {
 			break;
 		}
+		else
+		{
```

Comment:
Re "What's all this (below) do"

It's supposed to convert a keystroke into a chat command. Looking at the code, I'm not sure that it's in the right location. Perhaps the patch applied badly... Will look tomorrow.